### PR TITLE
Adding more objects to the docs.

### PR DESCRIPTION
**Pull request by maxhutch**
I heard you like docs @kjaym

### CODE REVIEW

File: docs/api/search/pif/query/core/Filter.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Description
+
+A Filter object describes conditions under which a field is matched.
+
+## Fields
+
+Field name | Value type | Default | Description
+-----------|------------|---------|------------
+`equal` | String |  | Exactly match |
+`min`   | Numeric | | Minimum bound (inclusive) |
```
Comment (kjaym) [inline docs/api/search/pif/query/core/Filter.md]: Min and max take strings. If the they can be interpreted as numbers then they are and a numeric filter is run. If they cannot be interpreted as numbers, then a string-based range is used (lexical comparison).

File: docs/api/search/pif/query/core/PropertyQuery.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Description
+
+A PropertyQuery object contains information about a desired property.
+
+## Fields
+
+Field name | Value type | Default | Description
+-----------|------------|---------|------------
+`name` | [`FieldOperation`](api/search/pif/query/core/FieldOperation) |  | Acceptable names of the property as a field operation |
```
Comment (kjaym) [inline docs/api/search/pif/query/core/PropertyQuery.md]: We should have a better description for fields that use a FieldOperation. Something like "Operations to execute against the name field." It doesn't necessarily need to be a filter, but the current description implies that.

Comment (maxhutch) [reply docs/api/search/pif/query/core/PropertyQuery.md]: I don't really get these either.  I assume `FieldOperation` could, at some point, contain more than `filter` and `extract_as`?

Comment (kjaym) [reply docs/api/search/pif/query/core/PropertyQuery.md]: That's right. It is just a bag for anything that could execute against the particular field.

**Pull request merged**